packages/shared/routes/dashboard/developerTools/DeveloperTools.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="typescript">
+    import { selectedAccount } from '@core/account'
+    import { localize } from '@core/i18n'
+    import { openPopup } from '@lib/popup'
+    import { OnboardingButton, Pane, Text } from 'shared/components'
+
+    function handleMintNativeTokenClick() {
+        openPopup({
+            type: 'mintNativeTokenForm',
+            overflow: true,
+        })
+    }
+</script>
+
+{#if $selectedAccount}
+    <div
+        class="w-full h-full flex flex-nowrap p-8 relative flex-1 bg-gray-50 dark:bg-gray-900 justify-center items-start"
+    >
+        {#key $selectedAccount?.id}
```

Comment:
why `key`?

Reply:
ok i see that its useful on the `wallet` tab, but here it doesn't do much, but we can leave it just in case

Reply:
it will destroy and remount whats betwen the `#key /key` tags when `$selectedAccount?.id` changes